cert-data.yml

```diff
@@ -64,6 +82,9 @@
 Default/OpenEdX/Certificate: &certificate_defaults
   VERSION: 4_programmatic
   TEMPLATEFILE: False
+  font: 
+    name: "SourceSansPro"
+    type: "Regular" 
```

Comment:
^ this is actually not true, but we should still change it... SourceSans will go the way of the dodo in this repo once the v4_programmatic renderer takes over...